lib/vanagon/component/source/git.rb

```diff
@@ -45,7 +45,7 @@ def initialize(url, workdir:, **options)
           # Ensure that #url returns a URI object
           @url = URI.parse(url.to_s)
           @ref = opts[:ref]
-          @workdir = workdir
+          @workdir = File.realpath(workdir) if workdir
```

Comment:
Yeah, traced through the codepaths and this will always be `@workdir` from the vanagon driver. Updating!